kalamine/server.py

```diff
@@ -10,8 +10,8 @@
 from .layout import KeyboardLayout, load_layout
 
 
-def keyboard_server(file_path: Path) -> None:
-    kb_layout = KeyboardLayout(load_layout(file_path))
+def keyboard_server(file_path: Path, angle_mod: bool) -> None:
+    kb_layout = KeyboardLayout(load_layout(file_path), angle_mod)
```

Comment:
I think it would make sens to force the ISO layout (for `kalamine watch`) when the `angle_mod` option is set.

Reply:
Good idea